src/component/base/DraftEditorProps.js

```diff
@@ -60,10 +60,10 @@ export type DraftEditorProps = {
   // For a given `ContentBlock` object, return an object that specifies
   // a custom block component and/or props. If no object is returned,
   // the default `DraftEditorBlock` is used.
-  blockRendererFn?: (block: ContentBlock) => ?Object,
+  blockRendererFn: (block: ContentBlock) => ?Object,
```

Comment:
What if a user doesn't pass this into Draft? And same question about the props below.

Reply:
The reason why we were able to delete it was that we already supply a default empty function for it https://github.com/facebook/draft-js/blob/master/src/component/base/DraftEditor.react.js#L73